### ISSUE TITLE
not work on 1.7.2.4

### ISSUE THREAD

**Issue by saostad** [tawk/tawk-prestashop#7]
Hi,
I trying to use your module on ver 1.7.2.4 but it doesn't show up for me.
I just could login with my account!

**Comment by afigienas** [comment]
Issue is that during installation module doesn't register the footer hook. To fix, need to open file tawkto.php, go to line 65. Instead:
```
return parent::install() && $this->registerHook('footer') && $this->installTab();
```
Change to this:
```
return parent::install() && $this->registerHook('displayFooter') && $this->installTab();
```

**Comment by Torrentus** [comment]
Unfortunately, that did not work for 1.7.5, also tried displayBeforeBodyClosingTag, no succes.

**Comment by jimix974** [comment]
hello, Anyone could use it on prestashop 1.7.4 please? I try to change the registerhook, but no success to.

thanks

**Comment by eug-L** [comment]
This issue has been fixed in the latest version for Prestashop 1.7
